src/fosslight_dependency/package_manager/Pypi.py

```
@@ -269,18 +266,15 @@ def start_pip_inspect(self):
 
-        env = os.environ.copy()
-        env['PYTHONUTF8'] = '1'
-
         command_list = [activate_command, pip_list_command, deactivate_command]
         command = command_separator.join(command_list)
 
         exists_pipdeptree = False
         try:
-            cmd_ret = subprocess.run(command, shell=True, env=env, stderr=subprocess.PIPE)
-            if cmd_ret.returncode != 0:
+            cmd_ret = subprocess.call(command, shell=True)
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)